carousel/docs/tutorials/tutorial_1.rst

```diff
@@ -36,15 +36,13 @@ power performance model, copy the following snippet into the new module::
         """
         Outputs for PV Power demo
         """
-        timestamps = {"isconstant": True, "size": 8761}
-        hourly_energy = {
-            "isconstant": True, "timeseries": "hourly_timeseries",
-            "units": "W*h",
-            "size": 8760
-        }
-        hourly_timeseries = {"isconstant": True, "units": "W*h", "size": 8760}
-        monthly_energy = {"isconstant": True, "units": "W*h", "size": 12}
-        annual_energy = {"isconstant": True, "units": "W*h"}
+        timestamps = OutputParameter(isconstant = True, size = 8761)
```

Comment:
need to add `from carousel.core.outputs import OutputParameter`, ditto for other tutorials